### ISSUE TITLE
False positive invalid-unary-operand-type for return value of np.diff()

### ISSUE THREAD

**Issue by eniemela3** [PyCQA/pylint#2954]
### Steps to reproduce
1. Create a file, say `file.py` with the following contents:
```
import numpy as np

var = -np.diff(np.array([1, 2.5, 5]))
print(type(var))
```
2. Run `pylint file.py`

### Current behavior
Pylint output contains `file.py:3:6: E1130: bad operand type for unary -: generator (invalid-unary-operand-type)`.

### Expected behavior
The type of the return value from np.diff should be recognized correctly as a ndarray, not a generator, and the error should not be reported.

### pylint --version output
```
pylint 2.3.1
astroid 2.2.5
Python 3.7.1 (v3.7.1:260ec2c36a, Oct 20 2018, 14:05:16) [MSC v.1915 32 bit (Intel)]
```

**Comment by hippo91** [comment]
@eniemela3  thank you for the report.
It should have been corrected through PyCQA/astroid#664.
Can you confirm please? Thanks again.
